src/app/dashboard/user-info/user-info.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+import {activityType} from "../../models/app.model";
+import {AppManager} from "../../managers/app.manager";
+
+@Component({
+  selector: 'app-user-info',
+  templateUrl: './user-info.component.html',
+  styleUrls: ['./user-info.component.scss']
+})
+export class UserInfoComponent implements OnInit {
+
+  constructor(private appManager: AppManager) { }
+
+  ngOnInit(): void {
+  }
+  activityType: number = activityType.daily;
```

Comment:
свойство `activityType` теперь не `number`, а одно из возможных в енуме `ActivityType`, т.е должно быть:
```suggestion
  activityType: ActivityType = activityType.daily;
```